bin/node/runtime/pangolin/src/pallets/ethereum_issuing.rs

```diff
@@ -9,7 +9,8 @@ frame_support::parameter_types! {
 }
 
 impl Config for Runtime {
-	type PalletId = EthereumIssuingPalletId;
+	// FIXME: Remove tight couple with dvm_ethereum and change it to PalletId
+	type IssuingPalletId = EthereumIssuingPalletId;
```

Comment:
why not use `PalletId` since it is convention?

Reply:
The `ethereum_issuing` has a tight couple with the `dvm_ethereum` pallet Config.  Once we gave `dvm_ethereum` pallet a config field `PalletId`, the `ethereum_issuing` pallet will show the following compile error:
```rs
pub trait Config: dvm_ethereum::Config {
 ....
}
```
```sh
error[E0221]: ambiguous associated type `PalletId` in bounds of `T`
  --> frame/bridge/ethereum/issuing/src/lib.rs:76:1
   |
76 | #[frame_support::pallet]
   | ^^^^^^^^^^^^^^^^^^^^^^^^
   | |
   | ambiguous associated type `PalletId`
   | help: use fully qualified syntax to disambiguate: `<T as pallet::Config>::PalletId`
...
86 |         type PalletId: Get<PalletId>;
   |         ----------------------------- ambiguous `PalletId` from `pallet::Config`
```

A simpler way for this pr to solve this problem is renaming `PalletId` in *_issuing pallet. I will raise another pull request to release the couple between *_issuing pallet and dvm_ethereum pallet.

Reply:
```
	pub trait Config: dvm_ethereum::Config {
		type Event: From<Event<Self>> + IsType<<Self as frame_system::Config>::Event>;
		#[pallet::constant]
		type PalletId: Get<PalletId>;
		type EthereumRelay: EthereumReceipt<Self::AccountId, RingBalance<Self>>;
		type EcdsaAuthorities: RelayAuthorityProtocol<Self::BlockNumber, Signer = EthereumAddress>;
		type WeightInfo: WeightInfo;
	}
```

@xiaoch05 It is confusing that why ethereum_issuing::Config has to inherit `dvm_ethereum::Config`, prefer to fix this FIXME in the same PR rather delay to future PR.

Reply:
OKay. I will push the fix later.